src/collections/blog/2023/2023-08-07-layer5-badge-program/index.mdx

```diff
@@ -0,0 +1,174 @@
+</div>
+<div>
+    <img src="https://github.com/layer5labs/meshery-extensions-packages/raw/master/assets/badges/first-collaborator/first-collaborator.svg" />
+    <span>GitOps with Friends</span> <p>This badge is awarded the first time a collaborator saves changes to one of your designs.</p>
+</div>
+</div>
+
+<h3>Project and Contribution Badges</h3>
+
+<div class="badge-grid">
+<div>
+  <img src="https://raw.githubusercontent.com/layer5labs/meshery-extensions-packages/master/assets/badges/ui-ux/ui-ux.svg" />
+  <span>UI/UX</span> <p>This badge is awarded to the community members who create or improve designs for visual aspects or user flow for any of the websites, flyers, promotions, Meshery UI, and so on in recognition and appreciation of their efforts.</p>
+</div>
+<div>
+  <img src="https://raw.githubusercontent.com/layer5labs/meshery-extensions-packages/master/assets/badges/meshmap/meshmap.svg" />
+  <span>MeshMap</span> <p>This badge is awarded to the community members who make consistent and impactful contributions to the <Link to="/cloud-native-management/meshmap">MeshMap</Link> project in recognition and appreciation of their efforts. Community members who earn this badge occasionally become a project maintainer.</p>
+</div>
+<div>
+  <img src="https://raw.githubusercontent.com/layer5labs/meshery-extensions-packages/master/assets/badges/smp/smp.svg" />
+  <span>Service Mesh Performance</span> <p>This badge isawarded to the community members who make consistent and impactful contributions to the Service Mesh Performance project. Community members who earn this badge occasionally become a project maintainer.</p>
+</div>
+<div>
+  <img src="https://raw.githubusercontent.com/layer5labs/meshery-extensions-packages/master/assets/badges/community/community.svg" />
+  <span>Community</span> <p>This badge is awarded to the community members who repeatedly engage in welcoming, encouraging, and supporting other Layer5 community members. Community members who earn this badge occasionally graduate to undertaking the Community Manager role.</p>
+</div>
+<div>
+  <img src="https://raw.githubusercontent.com/layer5labs/meshery-extensions-packages/master/assets/badges/writer-program/writer-program.svg" />
+  <span>Writer's Program</span> <p>awarded to the community members who make with two or more published writings whether in article, blog post, project documentation or other form in recognition and appreciation of their efforts.</p>
+</div>  
+<div>
+    <img src="https://raw.githubusercontent.com/layer5labs/meshery-extensions-packages/master/assets/badges/meshery/meshery.svg" />
+    <span>Meshery</span> <p>This badge isawarded to the community members who make consistent and impactful contributions to the Meshery project. Community members who earn this badge occasionally become a project maintainer.</p>
```

Comment:
Meshery

This badge is awarded to the community members who make consistent and impactful contributions to the Meshery project. Community members who earn this badge occasionally become project maintainers.

Reply:
Perfect! Thank you, @goldfishdolphin. You are most encouraged to "add a suggestion" in the form of a code update. Please do so at-will.

<img width="206" alt="Screenshot 2023-08-11 at 4 05 14 PM" src="https://github.com/layer5io/layer5/assets/7570704/ea62125e-2743-46aa-96f5-bbffbd2d615e">